src/platforms/node/guides/aws-lambda/index.mdx

```diff
@@ -80,6 +80,36 @@ exports.handler = Sentry.AWSLambda.wrapHandler(yourHandler, {
 });
 ```
 
+
+## `Promise.allSettled` Returned by the Handler
+_(New in version ??)_
```

Comment:
If there's no version set, we shouldn't add this line.
```suggestion
```

Reply:
There will be a version, right? For the moment I don't know it, but the feature didn't exist before? It's better not to put anything in this case ? 

https://github.com/getsentry/sentry-docs/pull/4374#issuecomment-964273769

Reply:
Yup it'll be with the next release, we can just update this PR when that goes out.